epxresults/job.py

```diff
@@ -273,54 +278,45 @@ def get_job_state_table(
 
         # check `count_type` keyword argument
         if count_type not in _count_types.keys():
-            msg = (f"`count_type` must be one of {_count_types.keys()}")
+            msg = f"`count_type` must be one of {_count_types.keys()}"
             raise ValueError(msg)
 
         prefix = _count_types[count_type]
         fname = f"{condition}.{prefix}{state}.csv"
         path_to_file = os.path.join(
-            self.path_to_job,
-            self._interval_path_component(interval),
-            fname
+            self.path_to_job, self._interval_path_component(interval), fname
         )
 
         # check if condition is available
         if condition not in self.conditions:
-            msg = (f"{condition} is not available to load. "
-                   "See `self.conditions` for a list of available conditions.")
+            msg = (
+                f"{condition} is not available to load. "
+                "See `self.conditions` for a list of available conditions."
+            )
             raise ValueError(msg)
 
         if not os.path.isfile(path_to_file):
-            msg = (f"'{path_to_file}' not found. `state`={state} may not be "
-                   f"a valid state in the FRED condition '{condition}'. "
-                   "Alternatively, the output for the requested interval "
-                   "may not be turned on.")
+            msg = (
+                f"'{path_to_file}' not found. `state`={state} may not be "
+                f"a valid state in the FRED condition '{condition}'. "
+                "Alternatively, the output for the requested interval "
+                "may not be turned on."
+            )
             raise ValueError(msg)
 
-        pattern = re.compile("RUN[0-9]+")
-        df = pd.read_csv(path_to_file, usecols=lambda x: pattern.match(x))
-        df.index.name = 'sim_day'
-
-        if format == 'long':
-            df['sim_day'] = df.index
-            df = pd.wide_to_long(df, ["RUN"], i="sim_day", j="run")
-            df = df.rename(columns={"RUN": count_type})
-            df = df.swaplevel().sort_index().reset_index()
-        elif format == 'wide':
-            # do nothing
-            pass
-        else:
-            msg = (f"Requested DataFrame format, {format}, not recognized.")
-            raise ValueError(msg)
+        df = self._read_plot_data_file(path_to_file, interval)
+        self._validate_table_format(format)
+        if format == "long":
+            df = self._convert_wide_plot_table_to_long(df, count_type)
 
         return df
 
     def get_job_variable_table(
-            self,
-            variable: str,
-            interval: str = 'daily',
-            format: str = 'long'
-            ) -> pd.DataFrame:
+        self,
+        variable: str,
+        interval: OutputInterval = "daily",
```

Comment:
Why are double quotes needed/suggested here?

Reply:
They're not strictly necessary, but 
1. Double quotes are used in other parts of this function definition, and consistency is good
2. I am in the habit of running [black](https://github.com/psf/black) over changes I make to ensure removal of unnecessary whitespace etc, and black uses double quotes.